notebook/services/contents/fileio.py

```diff
@@ -119,6 +171,9 @@ class FileManagerMixin(object):
     log : logging.Logger
     """
 
+    use_atomic_writing = Bool(True, config=True, help="""flag to deactivate
+            atomic writing on slow (like networked) file system.""")
```

Comment:
The help for this shouldn't assume users know what atomic writing is - it should mention what having it enabled means, as well as why you might want to disable it.

It also needn't start "flag to...". Have a look at some of our other config help strings to see how we tend to write them.